### DISCUSSION TITLE
Missing 'async' keyword?

### DISCUSSION THREAD

**Discussion by Lun4m** [mrcjkb/rustaceanvim#371] [Q&A] [answered]
I'm not sure if I'm asking in the right place, point me in the right direction otherwise.

The problem I have is that the `async` keyword never shows up in the autocomplete menu. 
Is it exposed by the LSP like the other keywords are? Or is it a problem with the `cmp.nvim_lsp` source?

**Comment by mrcjkb** [comment, marked as answer]
Hey :wave: 

completions are provided by rust-analyzer, so you're best off asking there :smile:

**Reply by Lun4m** [reply]
Haha I realized as soon as I posted that rustacean.nvim is "just" the nvim wrapper of rust-analyzer 😅
